eq-author-api/migrations/dropDatatypeFieldFromPipedValues.js

```diff
@@ -0,0 +1,40 @@
+const { flatMap, get, set } = require("lodash");
+
+module.exports = function dropDatatypeFieldFromPipedValues(questionnaire) {
+  const regex = /\s?data-type="[^"]*"/g;
+
+  const sections = flatMap(questionnaire.sections, section => section);
+  const pages = flatMap(questionnaire.sections, section =>
+    flatMap(section.pages, page => page)
+  );
+
+  const PIPEABLE_PAGE_FIELDS = [
+    "title",
+    "description",
+    "guidance",
+    "additionalInfoContent",
+    "confirmation.title",
```

Comment:
The `totalTitle` field on a calculated page can also have answers piped into it.